build/checks/crosslinks.py

```diff
@@ -0,0 +1,27 @@
+import test, sys, re, json
+
+regex = r"{name}(?!</a>)"
+
+artists = []
+
+with open("./www/artists.json") as artf:
+    data = json.loads(artf.read())
+    for artist in data:
+        artists.append(artist["name"])
+
+class Crosslinks(test.Test):
```

Comment:
I haven't looked at the Test class. Does it provide the definition of run()? Is this how one extends a class in Python?

Reply:
Yes